CMakeLists.txt

```diff
@@ -116,76 +153,91 @@ add_subdirectory(tests/src)
 
 enable_testing()
 
+# FIXME: Run custom clang tests until we can use LLVM's clang.cindex.
```

Comment:
What are the criteria for being able to switch back to LLVM's `clang.cindex`? We should capture those here.

Reply:
Added comment.